client/src/sections/material/EmbedLinkForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment } from 'react';
+import { type SubmitHandler, useForm } from 'react-hook-form';
+import type { IsActiveProps, MaterialFormInput } from '@/src/shared/utils';
+import RFInputField from '@/src/shared/components/ReactForm/RFInputField';
+import DivCollapse from '@/src/shared/components/Collapse/DivCollapse';
+import RFTextField from '@/src/shared/components/ReactForm/RFTextField';
+import RFSelectField from '@/src/shared/components/ReactForm/RFSelectField';
+import Button from '@/src/shared/components/Button';
+
+const EmbedLinkForm: React.FunctionComponent<IsActiveProps> = ({
+  isActive
+}: IsActiveProps) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<MaterialFormInput>();
+
+  const onSubmit: SubmitHandler<MaterialFormInput> = async (
+    data: MaterialFormInput
+  ): Promise<void> => {
+    console.log(data);
+  };
+
+  const folders = [
+    { value: '0', label: 'Nothing Selected' },
+    { value: '1', label: 'Example Folder 1' },
+    { value: '2', label: 'Example Folder 2' },
+    { value: '3', label: 'Example Folder 3' }
+  ];
+
+  const categories = [
+    { value: '0', label: 'Nothing Selected' },
+    { value: '1', label: 'Example Category 1' },
+    { value: '2', label: 'Example Category 2' },
+    { value: '3', label: 'Example Category 3' }
+  ];
+
+  const courses = [
+    { value: '0', label: 'Nothing Selected' },
+    { value: '1', label: 'Example Course 1' },
+    { value: '2', label: 'Example Course 2' },
+    { value: '3', label: 'Example Course 3' }
+  ];
+
+  return (
+    <Fragment>
+      <div
+        className={`p-4 max-h-[800px] overflow-y-scroll ${
```

Comment:
```suggestion
        className={`p-4 max-h-[800px] overflow-y-auto ${
```
Even the section does not have scrollable height its showing the scroll bar area, please apply this instead for all related
![image](https://user-images.githubusercontent.com/110364690/235052119-486f14a3-a02f-4f30-8032-e9b0c54c6b25.png)